Sources/Administrator/InputHandler.cpp

```diff
@@ -101,22 +154,304 @@ void InputHandler::sendProcess(const string& full_command)
   }
 
   string process_name = full_command.substr(full_command.find_first_of(" ")+1);
```

Comment:
if path to file contains whitespaces it's working fine cuz we are not using find_first_of(" ") for path to file.